pkg/scheduler/framework/interface.go

```diff
@@ -608,6 +609,25 @@ type Handle interface {
 	Parallelizer() parallelize.Parallelizer
 }
 
+// PreFilterResult wraps needed info for scheduler framework to act upon PreFilter phase.
+type PreFilterResult struct {
+	// The set of nodes that should be considered downstream; if nil then all nodes are eligible.
+	NodeNames sets.String
```

Comment:
If we take the intersection, if `len(NodeNames) == 0` it means that all nodes are ineligible, right?
And if PreFilterResult is nil then all nodes are eligible